controller/authenticationController.js

```diff
@@ -0,0 +1,67 @@
+const jwt = require("jsonwebtoken");
+const User = require("../models/userschema");
+const bcrypt = require("bcrypt");
+
+async function services(user) {
```

Comment:
I think generateAccessToken can directly be used
Else renaming it would be better